core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -140,6 +140,10 @@ abstract class KafkaServerTestHarness extends ZooKeeperTestHarness {
   def createTopic(topic: String, partitionReplicaAssignment: collection.Map[Int, Seq[Int]]): scala.collection.immutable.Map[Int, Int] =
     TestUtils.createTopic(zkClient, topic, partitionReplicaAssignment, servers)
 
+  def createTopic(topic: String, partitionReplicaAssignment: collection.Map[Int, Seq[Int]],topicConfig: Properties): scala.collection.immutable.Map[Int, Int] =
```

Comment:
The `topicConfig` parameter is not used, so the whole test does not really pick up your `topicConfig`

Reply:
Thanks for catch this. fixed.